src/extensions/default/bramble/lib/ConsoleManagerRemote.js

```diff
@@ -1,28 +1,70 @@
 (function(transport, global, console) {
     "use strict";
-
-    function _log(s){
-        transport.send("bramble-console", s);
+    
+    function transportSend(args, type) {        
+        // To avoid Runtime.evaluate messages
+        if(args[0] !== "Runtime.evaluate") {
+            transport.send("bramble-console", args, type);
+        }
     }
+    
+    // Implement console.log replacement
```

Comment:
@Pomax, I've tried to simplify by using yours as an example, and the only way which allows for the console functions to still register is the following:
```js
    function generateRouter(fname) {
        var args = Array.from(arguments).slice();
        transportSend(args, fname);
    }
    
    // Implement console.log replacement
    console.log = function() {
        generateRouter("log");
    }
```
That being said, the `args` sent are `fname` alone:
```js
console.log("Hello Developers"); 
```
Would log `log` into the console. Likewise for the other functions.

Using the `forEach` that you suggested above essentially triggers each of the console functions in the array on loading of the file in the browser. Which is why I tried the method I included above. I'm curious if you have a better implementation, or perhaps we should leave the version which though longer, seems to work with the different arguments. 

Again, just speculating so if I have mentioned anything which is incorrect, I'm open to learning from said mistake.